Reading List/Reading List/BookController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  BookController.swift
+//  Reading List
+//
+//  Created by Cora Jacobson on 6/6/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+class BookController {
+    
+    init() {
+        loadFromPersistentStore()
+    }
+    
+    var books: [Book] = []
+    
+    func createBook(_ title: String, _ reasonToRead: String) {
+        let book = Book(title, reasonToRead)
+        books.append(book)
+        saveToPersistentStore()
+    }
+    
+    func deleteBook(_ book: Book) {
+        if let bookIndex = books.firstIndex(of: book) {
+            books.remove(at: bookIndex)
+        }
+        saveToPersistentStore()
+    }
+    
+    func updateHasBeenRead(_ book: Book) {
+        if let bookIndex = books.firstIndex(of: book) {
+            switch books[bookIndex].hasBeenRead {
+            case true:
+                books[bookIndex].hasBeenRead = false
+            case false:
+                books[bookIndex].hasBeenRead = true
+            }
```

Comment:
This certainly works, just letting you know that there is a succinct way to do this:

``` swift
books[bookIndex].hasBeenRead.toggle()
```